e2e/provision/install_sandbox.sh

```diff
@@ -75,6 +75,14 @@ fact_caching_connection = /tmp
 EOT
 
 # Management cluster creation
+if [[ ${MGMT_CLUSTER_TYPE:-kind} == "kubeadm" ]]; then
```

Comment:
Silly question but do we get multiple mgmt clusters if this executes?

Reply:
If Google GKE or REd Hat decides to add their clusters to CI/CD, then we could get different types. For now it will just be kind and kubeadm.